packages/release-trigger/Dockerfile

```diff
@@ -14,9 +14,33 @@
 
 # Use a multi-stage docker build to limit production dependencies.
 
+FROM node:18.20.3-bullseye AS BASE
```

Comment:
Can we setup Renovate to suggest updates to this version?

Reply:
Can we do it separately? Also updating a bunch of images in #5365